charts/astronomer/templates/houston/api/houston-configmap.yaml

```diff
@@ -207,6 +207,8 @@ data:
           # This is here for upgrading to 0.11.1+ of Airflow.
           keda:
             enabled: false
+            namespaceLabels:
+              name: {{ .Release.Name }}
```

Comment:
Just to be sure, we want Release.Name (as you have) for this and not Release.Namespace right?

Reply:
good point... @dimberman wdyt?

Reply:
Houston does it exactly like this. https://github.com/astronomer/astronomer/blob/master/charts/astronomer/templates/houston/api/houston-configmap.yaml#L176-L177

Reply:
Yep @danielhoherd that link represents labels that we want to apply to every airflow namespace. I believe the new usage here under KEDA is to tell the `pgbouncer` `NetworkPolicy` to allow traffic from the KEDA controller inside the platform namespace. It needs to run a query on the deployment dbs, so it connects to these to do so.